### ISSUE TITLE
Can't import a .sass module from an .scss module

### ISSUE THREAD

**Issue by jcreamer898** [jtangelder/sass-loader#73]
Because of...

https://github.com/jtangelder/sass-loader/blob/master/index.js#L128

I cannot import a `.sass` module within an `.scss` module. 

I have something like this...

```css
// main.scss
@import "_vars";

.element { color: $color; }
```

```css
// some/where/else/_vars.sass
$color: "red"
```

**Comment by jhnns** [comment]
Would it be possible if we didn't set the option? How is node-sass supposed to know, which syntax is used?

**Comment by jhnns** [comment]
Shipped with `v1.0.2`.

Importing a file with the other language style requires the extension to be set explicitly.
